.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,42 +1,35 @@
-<!--
-⚠️ We have stopped merging new PRs, for now, into the Strapi core.
-The reason is that we are developing a new architecture for both the admin panel and for the plugins.
-This new architecture will provide stability to the Strapi core as we approach the release of Beta.
-We appreciate and welcome all your contributions, but until further notice, please do not submit a PR as it will not be merged.
-Furthermore, your pull request will have to be rewritten for the new architecture.
--->
-
 <!--
 Hello 👋 Thank you for submitting a pull request.
 
-To help us to merge your PR please follow these bullet points:
-- You wrote documentation
-- You wrote tests
-- Refer to the issue you are closing in your PR description - fix #issue
-- Let us know if this PR is WIP or ready to merge
+To help us merge your PR, make sure to follow the instructions below:
+- Create or update the documentation.
+- Create or update the tests.
+- Refer to the issue you are closing in the PR description - fix #issue
+- Specify if the PR is in WIP (work in progress) state or ready to be merged
```

Comment:
Do we want to say: Specify in the title, if the PR is in a WIP (word in progrss) state or ready to be merged, e.g. title: [WIP] Update the docs